.github/workflows/build+test.yml

```diff
@@ -26,7 +26,7 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
-    - uses: dtolnay/rust-toolchain@1.82.0 # Also test our Rust MSRV here.
+    - uses: dtolnay/rust-toolchain@1.83.0 # Also test our Rust MSRV here.
```

Comment:
Already included in #3322.

Reply:
Will automatically go away when rebasing. Rebased.